devlab/tests/cleanup.py

```diff
@@ -169,8 +175,8 @@ def clean_fips(self):
             try:
                 self.neutronclient.delete_floatingip(ip['id'])
             except nt_exceptions.NeutronClientException as e:
-                print "Ip %s failed to delete: %s" % (
-                    ip['floating_ip_address'], repr(e))
+                LOG.warning("Ip %s failed to delete: %s",
+                            ip['floating_ip_address'], repr(e))
```

Comment:
Use `LOG.warning("Ip %s failed to delete", ip['floating_ip_address'], exc_info=True)`, it will output exception with full stack trace.